### PR TITLE
Djpadz

### PR DESCRIPTION
**Pull request by djpadz**
Two new formulae: hubcap 3.17 and moe 1.7

### CODE REVIEW

**Comment by jacknagel** [comment]
Are these formulae related? If not, please submit them separately.

File: Library/Formula/hebcal.rb [hunk 1]
```diff
@@ -0,0 +1,17 @@
+class Hebcal < Formula
+  homepage "https://github.com/hebcal/hebcal"
+  url "https://github.com/hebcal/hebcal/archive/v3.17.tar.gz"
+  sha1 "03549601660bb20cc8e0a1e03664ef4f1b2549d0"
```
Comment (jacknagel) [inline Library/Formula/hebcal.rb]: Use `sha256`

File: Library/Formula/hebcal.rb [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+
+  def install
+    system "./configure", "--prefix=#{prefix}", "ACLOCAL=aclocal", "AUTOMAKE=automake"
+    system "make", "install"
+  end
+
+  test do
+    system "hebcal"
```
Comment (jacknagel) [inline Library/Formula/hebcal.rb]: Use full path (`#{bin}/hebcal`) in the test

File: Library/Formula/moe.rb [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+  url "http://ftpmirror.gnu.org/moe/moe-1.7.tar.lz"
+  sha256 "33ba66f948353c105232e9c8b2da8dc7b6ddc8cdff4beb3e3f55d78bc3acf1bb"
+
+  def install
+    system "./configure", "--prefix=#{prefix}"
+    system "make", "install"
+  end
+
+  test do
+    system "${bin}/moe", "--version"
```
Comment (jacknagel) [inline Library/Formula/moe.rb]: `#{bin}`, not `${bin}`

File: Library/Formula/moe.rb [hunk 4]
```diff
@@ -0,0 +1,14 @@
+class Moe < Formula
+  homepage "https://www.gnu.org/software/moe/moe.html"
+  url "http://ftpmirror.gnu.org/moe/moe-1.7.tar.lz"
```
Comment (jacknagel) [inline Library/Formula/moe.rb]: Please add `mirror "https://ftp.gnu.org/pub/gnu/moe/moe-1.7.tar.lz"` below this line

**Comment by djpadz** [comment]
I'm unsure as to how to create two new pull requests with the formulas separately.  In any case, I did put in the changes you'd asked.  Thanks for the help!

**Comment by xu-cheng** [comment]
Thanks for the pull request! 🎉 Homebrew depends on contributions from community members like you and we're grateful for your support.

Next time, you should put each formulae into separated branches so you can create PR for each one.

**Pull request closed without merging**